core/iwasm/compilation/aot_emit_function.c

```diff
@@ -1568,26 +1563,10 @@ aot_compile_op_call_indirect(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
                                      + 16))
         goto fail;
 
-#if WASM_ENABLE_LAZY_JIT == 0
-    /* Load function pointer */
-    if (!(func_ptr = LLVMBuildInBoundsGEP2(comp_ctx->builder, OPQ_PTR_TYPE,
-                                           func_ctx->func_ptrs, &func_idx, 1,
-                                           "func_ptr_tmp"))) {
-        aot_set_last_error("llvm build inbounds gep failed.");
-        goto fail;
-    }
-
-    if (!(func_ptr = LLVMBuildLoad2(comp_ctx->builder, OPQ_PTR_TYPE, func_ptr,
-                                    "func_ptr"))) {
-        aot_set_last_error("llvm build load failed.");
-        goto fail;
-    }
-#else
```

Comment:
Same as above, code of if branch is for AOT compiler mode and eager JIT mode.
How about:
```C
bool is_lazy_jit = false;
#if WASM_ENABLE_LAZY_JIT != 0
if (comp_ctx->is_jit_mode) {
    is_lazy_jit = true;
}
#endif

if (!is_lazy_jit) {
    ...
}
else {
    ...
}
```